qa/run-standalone.sh

```diff
@@ -1,25 +1,116 @@
-#!/bin/sh -ex
+#!/usr/bin/env bash
+set -e
 
-if [ ! -e Makefile ]; then
+if [ ! -e Makefile -o ! -d bin ]; then
     echo 'run this from the build dir'
     exit 1
 fi
 
+if [ ! -d /tmp/ceph-disk-virtualenv -o ! -d /tmp/ceph-detect-init-virtualenv ]; then
+    echo '/tmp/*-virtualenv directories not built. Please run "make check" first.'
+    exit 1
+fi
+
 if [ `uname` = FreeBSD ]; then
     # otherwise module prettytable will not be found
     export PYTHONPATH=/usr/local/lib/python2.7/site-packages
     exec_mode=+111
+    KERNCORE="kern.corefile"
+    COREPATTERN="core.%N.%P"
 else
+    export PYTHONPATH=/usr/lib/python2.7/dist-packages
     exec_mode=/111
+    KERNCORE="kernel.core_pattern"
+    COREPATTERN="core.%e.%p.%t"
+fi
+
+function finish() {
+    if [ -n "$precore" ]; then
+        sudo sysctl -w ${KERNCORE}=${precore}
+    fi
+    exit 0
+}
```

Comment:
@dzafman 
Exiting here with `0` will return `OKE` which is probably no the correct information once you run this in a testing environment.